README.rst

```diff
@@ -51,21 +51,21 @@ Install with all optional dependencies::
 
     $ pip install networkx[all]
 
-For additional details, please see `INSTALL.rst`.
```

Comment:
I prefer to leave this as it was. The link is to the local repository. So you shouldn't have to use a web browser.